src/components/DefaultHeader.jsx

```diff
@@ -0,0 +1,17 @@
+import SearchBar from './SearchBar';
+import DialogOpenButton from './DialogOpenButton';
+
+const DefaultHeader = () => {
+    return (
+        <div className="default-header__container">
+            <div className="add-movie__container">
+                <DialogOpenButton dialogTitle={"Add Movie"} buttonText={"+ ADD MOVIE"}  onDialogFormSubmit={() => {}}/>
```

Comment:
```suggestion
                <DialogOpenButton dialogTitle={"Add Movie"} buttonText={"+ ADD MOVIE"}  handleSubmit={() => {}}/>
```

Reply:
default header is removed in module 7 https://github.com/SerhiiSmilianets/reactjs-global-mentoring-program/pull/7/files